randovania/games/fusion/logic_database/header.json

```diff
@@ -614,6 +614,14 @@
             "OpenHatchLockRando": {
                 "long_name": "Open Hatch Lock Rando",
                 "extra": {}
+            },
+            "AntiSoftlock": {
+                "long_name": "Softlock Patches Enabled",
+                "extra": {}
+            },
+            "BomblessPBs": {
```

Comment:
please add this resource to our lay PB template

Reply:
Added